include/bluetooth/mesh/scheduler_srv.h

```diff
@@ -84,8 +84,6 @@ struct bt_mesh_scheduler_srv {
 	};
 	/** Composition data model pointer. */
 	struct bt_mesh_model *model;
-	/** Composition data setup model pointer. */
-	struct bt_mesh_model *setup_mod;
```

Comment:
Looks like this is an optimization rather than a bug.

Reply:
Yep, I optimized the solution but `setup_mod` left not initialized without this optimization. That caused access to not initialized pointer and unpredictable behavior.